draft-qproto-spec.md

```diff
@@ -1,965 +1,965 @@
-
-The `stream_id` field may be used to indicate that a specific stream will no longer
-receive any packets, and implementations are free to unload decoding and free up
-any used resources.
-
-The `stream_id` MAY be reused afterwards, but this is not recommended.
-
-If not encountered in a stream, and the connection was cut, then the receiver is
-allowed to gracefully wait for a reconnection.
-
-If encountered in a file, the implementation MAY regard any data present afterwards
-as padding and ignore it. Qproto files **MAY NOT** be concatenated.
-
-Streaming
-=========
-
-UDP
----
-To adapt Qproto for streaming over UDP is trivial - simply send the data packets
-as-is specified, with no changes required. The sender implementation SHOULD
-resent [session start](#session-start-packets),
-[time synchronization](#time-synchronization-packets) packets, as well as
-all stream registration, initialization and video info packets for all streams
-at most as often as the stream with the lowest frequency of `keyframe`s in order
-to permit for implementations that didn't catch on the start of the stream begin
-decoding.
-UDP mode is unidirectional, but the implementations are free to use the
-[reverse signalling](#reverse-signalling) data if they negotiate it themselves.
-
-Implementations MUST segment the data such that the network MTU is never
-exceeded and no packet fragmentation occurs.</br>
-The **minimum** network MTU required for the protocol is **384 bytes**,
-as to allow [video information packets](#video-info-packets) or any future large
-packets to be sent without fragmentation.
-
-Jumbograms MAY be used where supported to reduce overhead and increase efficiency.
-
-Data packets MAY be padded by appending random data or zeroes after the `packet_data`
-field up to the maximum MTU size. This permits constant bitrate operation,
-as well as preventing metadata leakage in the form of a packet size.
-
-UDP-Lite
---------
-UDP-Lite (IETF RFC 3828) SHOULD be preferred to UDP, if support for it is
-available throughout the network.</br>
-When using UDP-Lite, the same considerations as [UDP](#udp) apply.
-
-As UDP-Lite allows for variable checksum coverage, the **minimum** checksum
-coverage MUST be used, where only the header (8 bytes) is checksummed.
-
-QUIC
-----
-Qproto tries to use as much of the modern conveniences of QUIC (IETF RFC 9000)
-as possible. As such, it uses both reliable and unreliable streams, as well
-as bidirectionality features of the transport mechanism.
-
-All data packets with descriptors `0x01**`, `0xff`, `0xfe`, `0xfd` and `0xfc`
-MUST be sent over in an *unreliable* QUIC DATAGRAM stream, as per
-IETF RFC 9221.</br>
-All other packets MUST be sent over a reliable steam. FEC data for those packets
-SHOULD NOT be signalled.
-
-Implementations MUST segment the data such that the network MTU is never
-exceeded and no packet fragmentation occurs.</br>
-The **minimum** network MTU required for the protocol is **384 bytes**,
```

Comment:
I counted VIP as `148 bytes`. UDP protocol only has small overhead. Is your `384 bytes` not based on an exact value or am I missing something? (I haven't inspected all the tables and numbers in detail, as you will understand.)